network/stellar-disbursement-platform/admin-guide/migrating-to-sdp-multi-tenant.mdx

```diff
@@ -1,89 +1,221 @@
+- `CHANNEL_ACCOUNT_ENCRYPTION_PASSPHRASE`: A Stellar-compliant ed25519 private key used to encrypt/decrypt the channel accounts' private keys. When not set, it will default to the value of the 'distribution-seed' option, which was used in the single-tenant version. **Attention**, when migrating from the single-tenant, setting this config to something different from the old `DISTRIBUTION_SEED` will prevent the code from being able to decrypt the channel accounts.
+- `DISTRIBUTION_SIGNER_TYPE`: The type of signer used to sign Stellar transactions for the tenants' distribution accounts. Options: `DISTRIBUTION_ACCOUNT_DB` (recommended), `DISTRIBUTION_ACCOUNT_ENV` (same as the single-tenant version).
+- `DISTRIBUTION_ACCOUNT_ENCRYPTION_PASSPHRASE`: A Stellar-compliant ed25519 private key used to encrypt/decrypt the in-memory distribution accounts' private keys. It's mandatory when the distribution-signer-type is set to `DISTRIBUTION_ACCOUNT_DB`.
 
-Scheduler environment variables: 
+Event broker configuration environment variables:
+
+- `BROKER_URLS`: A comma-separated list of the message broker URLs.
+- `CONSUMER_GROUP_ID`: Specifies a group ID for the broker consumers.
+- `EVENT_BROKER_TYPE`: Specifies the type of event broker to be used. Options: "KAFKA", "NONE". Defaults to "Kafka".
+- `KAFKA_SECURITY_PROTOCOL`: Defines the security protocol for Kafka. Options: PLAINTEXT, SASL_PLAINTEXT, SASL_SSL, SSL.
+- `KAFKA_SASL_USERNAME`: Specifies the Kafka SASL Username, required when the kafka security protocol is set to either `SASL_PLAINTEXT` or `SASL_SSL`.
+- `KAFKA_SASL_PASSWORD`: Specifies the Kafka SASL Password, required when the kafka security protocol is set to either `SASL_PLAINTEXT` or `SASL_SSL`.
+- `KAFKA_SSL_ACCESS_KEY`: The Kafka Access Key (keystore) in PEM format, required when the kafka security protocol is set to `SSL`.
+- `KAFKA_SSL_ACCESS_CERTIFICATE`: The Kafka SSL Access Certificate in PEM format that matches with the Kafka Access Key, required when the kafka security protocol is set to `SSL`.
+
+Scheduler environment variables:
 
 - `ENABLE_SCHEDULER`: Default "false". This enables scheduled jobs that replace the operations of a broker for synchronizing payments between SDP and TSS as well as submitting receiver invitations.
 - `SCHEDULER_PAYMENT_JOB_SECONDS`: Interval in seconds for the job that synchronizes payments between SDP and TSS. Minimum is 5s.
 - `SCHEDULER_RECEIVER_INVITATION_JOB_SECONDS`: Interval in seconds for the job that submits receiver invitations. Minimum is 5s.
 
 On the Anchor Platform side, we must set the following envs:
 
-- `SEP10_HOME_DOMAINS`: "localhost:8000, *.stellar.local:8000" # Comma separated list of home domains
-- `SEP10_HOME_DOMAIN`: "" # Here should be empty for the Multi-tenant version
+- `SEP10_HOME_DOMAINS="localhost:8000, *.stellar.local:8000"`: a comma-separated list of home domains used for [SEP-10].
+- `SEP10_HOME_DOMAIN=""`: this should be an empty string for the Multi-tenant version.
```

Comment:
I believe we need to add `SEP10_WEB_AUTH_DOMAIN` here

Reply:
Addressed in https://github.com/stellar/stellar-docs/pull/496/commits/143409eb702fceebac7b7cf0b6c744e10ccd3a14